docs/read_and_store_replay.ipynb

```diff
@@ -62,7 +62,9 @@
     "        for fhr in forecast_hours:\n",
     "            files.append(\n",
     "                    f\"{fp}{date.year:04d}{date.month:02d}{date.day:02d}{date.hour:02d}_fhr{fhr:02d}_control\")\n",
-    "    return [join(upper, this_dir, this_file) for this_file in files]"
+    "    # KJW: Do not use OS specific delimiters for URLs\n",
+    "    return [f\"{upper}/{this_dir}/{this_file}\" for this_file in files]\n",
+    "    # return [join(upper, this_dir, this_file) for this_file in files]\n"
```

Comment:
Is this a problem for accessing data on s3 from windows?

Reply:
It is when you use os.path.join.  On Windows, that makes the separator '\' and on unix a '/'.  URLs want '/' only.  This is why I had my issue yesterday with the cryptic key not found error.  I run on a Windows box so the S3 descriptor was mangled.